prov/efa/src/efa_user_info.c

```diff
@@ -302,7 +302,7 @@ bool efa_user_info_should_support_hmem(int version)
 	if (hmem_ops[FI_HMEM_CUDA].initialized && FI_VERSION_GE(version, FI_VERSION(1, 18))) {
 		EFA_INFO(FI_LOG_CORE,
 			"User is using API version >= 1.18. CUDA library and "
-			"device are available, claim support of FI_HMEM.");
+			"device are available, claim support of FI_HMEM.\n");
```

Comment:
minor, but since you have a whole commit for this change: s/device are/devices are/

Reply:
ack